Inception/Inception/Screens/SleepTracker/SleepTrackerTableViewController.swift

```diff
@@ -11,13 +11,23 @@ class SleepTrackerTableViewController: UITableViewController {
   
   /// 기록 없을 때 나오는 빈 화면
   @IBOutlet var recordEmptyView: UIView!
+  
+  /// 테스트 기록 추가 버튼
+  @IBAction func addTestRecord(_ sender: Any) {
+    saveNewItem(sleepRecordItem: SleepRecord(sleepSatisfacation: SleepSatisfacation.good, bedtimeDate: Date.now.addingTimeInterval(-60*60*6.5), wakeuptimeDate: Date.now))
```

Comment:
```suggestion
    let sleepRecordItem = SleepRecord(sleepSatisfacation: SleepSatisfacation.good, 
                                      bedtimeDate: Date.now.addingTimeInterval(-60 * 60 * 6.5)
    saveNewItem(sleepRecordItem: sleepRecordItem, wakeuptimeDate: Date.now))
```

p3;

연산자 사이에 띄어쓰기가 추가되면 피연산자가 잘 구분돼서 가독성이 좋아질 거 같아요!
혹시 띄어쓰기를 추가하는 것에 대해 어떻게 생각하실까요? 👀

또한, 라인이 많이 길어져서 `SleepRecord` 인스턴스 생성 코드를 별도의 선언으로 분리하고 개행을 추가해주면 한눈에 들어오기 쉬울 거 같아요~

그리고 `-60 * 60 * 6.5` 의 숫자가 무엇을 의미하는지 알 수 없는 숫자라(이런걸 매직넘버라고 합니다)
별도의 상수로 선언해주면 코드를 이해하는 데 도움될 거 같습니다!

Reply:
- 오 확실히 전보다 시원한 느낌이 듭니다!! 연산자에도 공백 추가..메모...

- **별도의 선언으로 분리** 라는 것이 이렇게 따로 상수를 만드는 것이 맞을까요? 😭

- 앗 보는 사람에 대한 배려가 부족했습니다.. 따로 상수로 빼서 이름을 붙여주겠습니다!

967b45eadbd2a491285d74fbd9c3654ebb24bafc

Reply:
**별도의 선언으로 분리** 에 대해 생각했던 코드는 https://github.com/DeveloperAcademy-POSTECH/MC3-Team13-Inception/commit/967b45eadbd2a491285d74fbd9c3654ebb24bafc 에 작성된 코드와 같은 내용이었습니다~
추가로 `testAtualSleepTime`에 타입 어노테이션을 추가해주면 타입추정을 줄일 수 있어 컴파일 성능 향상에 도움될 거 같아요!